static/javascript/index.js

```diff
@@ -171,6 +280,14 @@ function removeNoteFromDB(event){
 
 }
 
+/// Sign-Out with Facebook //////
+function sign_out(){
+    console.log("signed_out running")
+     window.location = "/";
+}
```

Comment:
When you delete a session even if you navigate backwards the session is still gone so the back button just brings you to the home page signed-out?